Release 2.4.0

andreriesco opened the pull request.

Commit:
dotnet: Update the SSH copy to COPY inside the container

Signed-off-by: Andre Riesco <andre.riesco@toradex.com>

Commit:
python3: Update the SSH copy to COPY inside the container

Signed-off-by: Andre Riesco <andre.riesco@toradex.com>

Commit:
zigConsole: Fix output not showing when debugging on the board

Fix output not showing on the debug console when debugging on the
board.

Signed-off-by: Andre Riesco <andre.riesco@toradex.com>

Commit:
torizon-base-cointainers:Bump the minor tag to the latest version(3.2.1)

Commit:
dotnet Uno: Fix .csproj path of the clean-debug- tasks

Signed-off-by: Andre Riesco <andre.riesco@toradex.com>

Commit:
README: Add the warning telling about the developer website documentation

Fixes:#140

Signed-off-by: Andre Riesco <andre.riesco@toradex.com>

Commit:
README: Remove the doc part about Extra Files (move to developer website)

Removing this section about working with extra files from the READMEs,
as it is going to be on the developer website.

Signed-off-by: Andre Riesco <andre.riesco@toradex.com>

microhobby committed:
monoCsharpForms: add dotnet.preferCSharpExtension

Mono for sure is something that is legacy and should never be used
again.

Signed-off-by: Matheus Castello <matheus.castello@toradex.com>

microhobby committed:
monoCsharpForms: Fix wrong use of USER

USER instruction does not have action in the build process, only
in the image already generated, so it is not possible to use
USER to change the owner of the next instructions.

Signed-off-by: Matheus Castello <matheus.castello@toradex.com>

microhobby committed:
dotnetAvalonia: Fix try to access the X11 display as root

Fix #159

Signed-off-by: Matheus Castello <matheus.castello@toradex.com>

Commit:
Dockerfile.debug: Add --chown=$SSHUSERNAME:$SSHUSERNAME to the COPY

Signed-off-by: Andre Riesco <andre.riesco@toradex.com>

Commit:
nodeElectron: Fix Dockerfile

Signed-off-by: Andre Riesco <andre.riesco@toradex.com>

microhobby commented on aspnetBlazor/.vscode/tasks.json:
```diff
@@ -138,20 +138,10 @@
             "detail": "",
             "hide": true,
-            "command": "scp",
+            "command": "sleep",
```
maybe is better use an `echo ok` than a `sleep 1`?

microhobby replied:
let's leave this one for this cycle, and the next one we need to fix to use the wait-a-bit instead and use an echo here.

microhobby commented:
LGTM!

microhobby committed:
dotnetAvaloniaFrameBuffer: update to .NET 8.0

Signed-off-by: Matheus Castello <matheus.castello@toradex.com>

microhobby committed:
dotnetSlint: Fix release container not finding ./ui/ folder

Signed-off-by: Matheus Castello <matheus.castello@toradex.com>

Commit:
dotnetAvalonia: Add Hello Torizon! mesage for automated tests

Signed-off-by: Andre Riesco <andre.riesco@toradex.com>

Commit:
cppSlint: Fix Dockerfile

Signed-off-by: Andre Riesco <andre.riesco@toradex.com>

Commit:
Dockerfile: Use WORKDIR ${APP_ROOT} instead of ENV APP_EXECUTABLE_PATH

Moving it back to the way the executable was being executed, as it is
the proper way to use it.

Signed-off-by: Andre Riesco <andre.riesco@toradex.com>

microhobby committed:
javaForms: Fix use of msopenjdk

There is no msopenjdk-jre package on the repo, so we need to use
msopenjdk-21 instead.

Signed-off-by: Matheus Castello <matheus.castello@toradex.com>

microhobby committed:
dotnetAvaloniaFrameBuffer: Fix production Dockerfile to use base 4-rc

Signed-off-by: Matheus Castello <matheus.castello@toradex.com>

Commit:
dotnetUnoFrameBuffer: Add Hello Torizon for automated test

Signed-off-by: Andre Riesco <andre.riesco@toradex.com>
